install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1613,142 +1613,145 @@ common issues:
+determined to be healthy.
 
-Once the green nodes are in `*Ready*` state, they can be made scheduleable. The
-blue nodes are then disabled so that no new pods are run on them.
+Once the green nodes are in *Ready* state, they can be made schedulable. Then,
+blue nodes are disabled so that no new pods are run on them.
 
 ----
-oadm manage-node --schedulable=true --selector=color=green
-oadm manage-node --schedulable=false --selector=color=blue
+# oadm manage-node --schedulable=true --selector=color=green
+# oadm manage-node --schedulable=false --selector=color=blue
 ----
 
 A common practice is to scale the registry and router pods until they are
-migrated to the green nodes.  For these pods, a canary deployment approach is
-commonly used.  Scaling them up will make them immediately active on the new
-nodes.  Pointing the deployment configuration to the new image causes a rolling
-update to happen.  However, because of node anti-affinity, and the fact that the
+migrated to the green nodes. For these pods, a canary deployment approach is
+commonly used. Scaling them up will make them immediately active on the new
+nodes. Pointing the deployment configuration to the new image causes a rolling
+update to happen. However, because of node anti-affinity, and the fact that the
 blue nodes are still unschedulable, the deployments to the old nodes will fail.
 At this point, the registry and router deployments can be scaled down to the
-original number of pods.  At any given point the original number of pods is
+original number of pods. At any given point, the original number of pods is
 still available so no capacity is lost.
 
+[[manual-upgrades-warming-the-new-nodes]]
 === Warming the New Nodes
 
 In order for pods to be migrated from the blue environment to the green, the
-images must be pulled.  Network latency and and load on the registry can cause
-delays if there is not sufficient capacity built in to the environment. Often
-the best way to minimize impact to the running system is trigger new pod
-deployments that will land on the new nodes.  This can be accomplished by
-importing new ImageStreams.
+images must be pulled. Network latency and load on the registry can cause delays
+if there is not sufficient capacity built in to the environment. Often, the best
+way to minimize impact to the running system is trigger new pod deployments that
+will land on the new nodes. Accomplish this by importing new *ImageStreams*.
 
-Quite often, a major release of {product-title} is the motivation for a blue-green
-deployment.  When this occurs, there will be new ImageStreams available for users of
-Source-to-Image.  Upon import, any builds or deployments configured with
-ImageChangeTriggers will be automatically created.
+Quite often, a major release of {product-title} is the motivation for a
+blue-green deployment. When this occurs, there will be new *ImageStreams*
+available for users of Source-to-Image. Upon import, any builds or deployments
```

Comment:
* "Quite often" seems noisy.
* Likewise, "When this occurs"

Maybe these two sentences can be smooshed somehow?